isobus/src/isobus_virtual_terminal_objects.cpp

```diff
@@ -8623,4 +8623,166 @@ namespace isobus
 		pointerType = type;
 	}
 
+	VirtualTerminalObjectType ScaledGraphicObject::get_object_type() const
+	{
+		return VirtualTerminalObjectType::ScaledGraphic;
+	}
+
+	bool ScaledGraphicObject::get_attribute(uint8_t attributeID, uint32_t &returnedAttributeData) const
```

Comment:
```suggestion
	bool ScaledGraphicObject::get_attribute(std::uint8_t attributeID, std::uint32_t &returnedAttributeData) const
```

Check out the other places in this file as well for this pls